packages/wonder-blocks-dropdown/components/multi-select.js

```diff
@@ -284,28 +306,84 @@ export default class MultiSelect extends React.Component<Props, State> {
         }
     }
 
-    getMenuItems(): Array<DropdownItem> {
-        const {children, selectedValues} = this.props;
-        return React.Children.toArray(children)
-            .filter(Boolean)
-            .map((option) => {
-                const {disabled, value} = option.props;
-                return {
-                    component: option,
-                    focusable: !disabled,
-                    populatedProps: {
-                        onToggle: this.handleToggle,
-                        selected: selectedValues.includes(value),
-                        variant: "checkbox",
-                    },
-                };
+    getMenuItems(
+        children: Array<React.Element<OptionItem>>,
+    ): Array<DropdownItem> {
+        const {isFilterable} = this.props;
+        // If it's not filterable, no need to do any extra besides mapping the
+        // option items to dropdown items.
+        if (!isFilterable) {
+            return children.map(this.mapOptionItemToDropdownItem);
+        }
+
+        const {searchText, lastSelectedValues} = this.state;
+
+        let filteredChildren = children;
```

Comment:
`let` is a code smell to me. I think this could be refactored by moving the creation of `filteredChildren` into a function so that `filteredChildren` is `const.